readthedocs/config/config.py

```diff
@@ -399,26 +396,22 @@ def validate_python(self):
                         'python.extra_requirements',
                         self.PYTHON_EXTRA_REQUIREMENTS_INVALID_MESSAGE,
                         code=PYTHON_INVALID)
-                for extra_name in raw_extra_requirements:
-                    with self.catch_validation_error(
-                            'python.extra_requirements'):
-                        python['extra_requirements'].append(
-                            validate_string(extra_name)
-                        )
+                if not python['install_with_pip']:
+                    python['extra_requirements'] = []
+                else:
+                    for extra_name in raw_extra_requirements:
+                        with self.catch_validation_error(
+                                'python.extra_requirements'):
+                            python['extra_requirements'].append(
+                                validate_string(extra_name)
+                            )
 
             # Validate setup_py_install.
             if 'setup_py_install' in raw_python:
                 with self.catch_validation_error('python.setup_py_install'):
-                    python['setup_py_install'] = validate_bool(
+                    python['install_with_setup'] = validate_bool(
                         raw_python['setup_py_install'])
 
-            # Validate setup_py_path.
-            if 'setup_py_path' in raw_python:
```

Comment:
Did we remove this option?

Reply:
In V1 we were validating that the path to `setup.py` exists and here we are removing this check.

Reply:
This option was never implemented in v1, we are considering implementing this in v2 #4354